docs/core/testing/unit-testing-visual-basic-with-tunit.md

```diff
@@ -0,0 +1,254 @@
+* Replace the code in *PrimeService.vb* with the following code:
+
+  ```vb
+  Imports System
+
+  Namespace Prime.Services
+      Public Class PrimeService
+          Public Function IsPrime(candidate As Integer) As Boolean
+              Throw New NotImplementedException("Not implemented.")
+          End Function
+      End Class
+  End Namespace
+  ```
+
+* The preceding code:
+  * Throws a <xref:System.NotImplementedException> with a message indicating it's not implemented.
+  * Is updated later in the tutorial.
+
+<!-- preceding code shows an english bias. Message makes no sense outside english -->
+
+* In the *unit-testing-using-dotnet-test* directory, run the following command to add the class library project to the solution:
+
+  ```dotnetcli
+  dotnet sln add ./PrimeService/PrimeService.vbproj
+  ```
+
+* Install the TUnit project template:
+
+  ```dotnetcli
+  dotnet new install TUnit.Templates
+  ```
+
+* Create the *PrimeService.Tests* project by running the following command:
+
+  ```dotnetcli
+  dotnet new tunit -o PrimeService.Tests -lang VB
+  ```
+
+* The preceding command:
+  * Creates the *PrimeService.Tests* project in the *PrimeService.Tests* directory. The test project uses [TUnit](https://tunit.dev/) as the test library.
+  * Configures the test runner by adding the `TUnit` package to the project file.
+
+* Add the test project to the solution file by running the following command:
+
+  ```dotnetcli
+  dotnet sln add ./PrimeService.Tests/PrimeService.Tests.vbproj
+  ```
+
+* Add the `PrimeService` class library as a dependency to the *PrimeService.Tests* project:
+
+  ```dotnetcli
+  dotnet add ./PrimeService.Tests/PrimeService.Tests.vbproj reference ./PrimeService/PrimeService.vbproj
+  ```
+
+<a name="create-test-cmd"></a>
+
+### Commands to create the solution
+
+This section summarizes all the commands in the previous section. Skip this section if you've completed the steps in the previous section.
+
+The following commands create the test solution on a Windows machine. For macOS and Unix, update the `ren` command to the OS version of `ren` to rename a file:
+
+```dotnetcli
+dotnet new sln -o unit-testing-using-dotnet-test
+cd unit-testing-using-dotnet-test
+dotnet new classlib -o PrimeService -lang VB
+ren .\PrimeService\Class1.vb PrimeService.vb
+dotnet sln add ./PrimeService/PrimeService.vbproj
+dotnet new install TUnit.Templates
+dotnet new tunit -o PrimeService.Tests -lang VB
+dotnet add ./PrimeService.Tests/PrimeService.Tests.vbproj reference ./PrimeService/PrimeService.vbproj
+dotnet sln add ./PrimeService.Tests/PrimeService.Tests.vbproj
+```
+
+Follow the instructions for "Replace the code in *PrimeService.vb* with the following code" in the previous section.
+
+## Configure Microsoft.Testing.Platform mode
+
+TUnit only supports Microsoft.Testing.Platform and doesn't support VSTest. To use `dotnet test` with TUnit, add the following configuration to your `global.json` file in the solution root:
```

Comment:
It's best IMO to let TUnit templates do this on its own.

Reply:
I'll make a note to update TUnit templates